src/autopas/particles/ParticleBase.h

```diff
@@ -152,6 +157,22 @@ class ParticleBase {
    */
   void setR(const std::array<double, 3> &r) { _r = r; }
 
+  /**
+   * update rebuild position of the particle
+   */
+  void setRAtRebuild() { _rAtRebuild = _r; }
+
+  /**
+   * calculate distance since last rebuild
+   * this is used to check if neighbour lists are still valid inside the logic handler
+   * @return displacement vector of particle since rebuild
```

Comment:
```suggestion
   * Calculate the distance since the last rebuild.
   * This is used to check if neighbor lists are still valid inside the logic handler
   * @return displacement vector of particle since rebuild
```
American English :P

Reply:
Done